mizar/store/operator_store.py

```diff
@@ -128,6 +133,14 @@ def delete_ep(self, name):
         l = len(self.eps_net_store[ep.net])
         if l == 0:
             del self.eps_net_store[ep.net]
+        
+        if ep.k8s_pod_name in self.eps_pod_store.keys():
+            if name not in self.eps_pod_store[ep.k8s_pod_name]:
+                return ep
+            self.eps_pod_store[ep.k8s_pod_name].pop(name)
+            l = len(self.eps_pod_store[ep.k8s_pod_name])
+            if l == 0:
+                del self.eps_pod_store[ep.k8s_pod_name]
```

Comment:
What is the use of that block? Why do we even need an eps_pod_store?

Reply:
This has to do with arktos renaming the pod. I notice that in arktos, say if we create a pod called pod1, arktos will append namespace and tenant name to the pod name. If nothing is specified, it will reassign the name follow by random hash.
I agree that this part of code needs modifications.